dtests/src/test/java/io/snappydata/hydra/testDMLOps/local.dmlOps.overflow.conf

```diff
@@ -0,0 +1,31 @@
+hydra.Prms-removeDiskFilesAfterTest = true;
+hydra.Prms-extraDerbyServerVMArgs += " -Xmx4096m -Dderby.storage.pageCacheSize=32000 -Dderby.locks.waitTimeout=60 -Dderby.locks.deadlockTimeout=40 ";
+hydra.Prms-extraDerbyServerVMArgs += " -XX:MaxPermSize=128m ";
+
+hydra.HostPrms-hostNames =
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${snappyStoreHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> \", ${leadHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${locatorHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${workerHosts})" ncf
+;
+
+hydra.VmPrms-extraVMArgs =
+  fcn
+       "hydra.TestConfigFcns.duplicate(\"-Xms3072m -Xmx3072m\", ${${A}Hosts}, true)"
+  ncf,
+  fcn
+       "hydra.TestConfigFcns.duplicate(\"-Xms2072m -Xmx2072m\", ${${B}Hosts}, true)"
+  ncf,
+  fcn
+       "hydra.TestConfigFcns.duplicate(\"-Xms1024m -Xmx1024m\", ${${C}Hosts}, true)"
+  ncf;
+
+io.snappydata.hydra.cluster.SnappyPrms-serverLauncherProps = " -heap-size=2g -memory-size=2g ";
+//io.snappydata.hydra.cluster.SnappyPrms-serverLauncherProps += " -J-Dgemfire.VersionVector.VERBOSE=true -J-Dgemfirexd.debug.true=TraceTran,TraceQuery ";
+io.snappydata.hydra.cluster.SnappyPrms-locatorLauncherProps = " ";
```

Comment:
Remove the locatorLauncherProps object as its unnecessary without having any param.